tests/test_middleware.py

```diff
@@ -244,3 +244,21 @@ async def handler(request):
             token.decode('utf-8')),
     })
     assert response.status == resp_status
+
+
+async def test_pass_preflight_options_request(create_app, aiohttp_client, fake_payload, token):
+    async def handler(request):
+        assert request.method == hdrs.METH_OPTIONS
```

Comment:
This check seems unnecessary. As it done on middleware side.

Reply:
Hi, fixed everything.